static/js/workshop.js

```diff
@@ -0,0 +1,20 @@
+// My dislike of jQuery is overshadowed by the ease of this use-case
+$(document).ready(function() {
+    var publicWorkshop = $('#id_public');
+    var privateCodeRow = $('.field-private_code');
+    var privateCodeInput = $('#id_private_code');
+    var privateCodeUrlSpan = $('#pcode');
```

Comment:
This would require quite a bit of hacking with JS. I tried to go the same route as the private code with the discount code but the help text just goes up to the question mark alt text next to the column header so I put a generic help text. I could try to get it to happen, but it will need a lot more lines than the private code.